code/src/components/app-button/AppButton.scss

```diff
@@ -1,40 +1,123 @@
 @import "@design-system";
 
-// @TODO: It is just an example. Remove it or modify later
-.button__contained {
-  // Example of using variable from design system
-  background-color: var(--spa-button-bg-color, $purple);
+@mixin button-size($padding, $font-size) {
+  padding: spacing(2) spacing($padding);
+  font-size: var(--spa-button-font-size, $font-size);
+}
+
+@mixin focus-style($color) {
+  &:focus {
+    outline: 2px solid var(--spa-button-focus-color, $color);
+    outline-offset: 2px;
+  }
+}
+
```

Comment:
This code pollutes modularity rule, because all buttons that use `focus-style` mixin use same css variable. Generally the variables should be modular, for example we should have ability to change color of all disabled buttons or just one button using css variables. I'd better refactor it. What do you think?

Same for other mixins, just try to pass css variable with colot, not just color

Reply:
Refactored this mixin